### PR TITLE
adds metal bats

### PR DESCRIPTION
**Pull request by PKPenguin321**
# UPDATED VIDEO: http://puu.sh/qqpsZ/5796be58f9.mp4

adds admin only ability to bats that allows them to hit home runs

adds a metal bat that reflects projectiles. supposed to reflect thrown items but the support for that doesn't really exist yet, as such it's admin only too. willing to remove it if maintainers think it's bloat (especially since it adds 2 sound files)

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
Sad that it's not the smash homerun sound

**Comment by PKPenguin321** [comment]
i woulda but i was like "copyright tho"
i'm actually not sure if there is copyright but if there isn't then i'll use it but you gotta find out for me ok

**Comment by optimumtact** [comment]
Nah we have enough bats, no more meme weapons thanks.

**Comment by PKPenguin321** [comment]
ok i figured

**Comment by optimumtact** [comment]
I just watched that vdeo and it's good so make the wood bat do that as an admin spawn thing.

**Comment by PKPenguin321** [comment]
oh that's actually a good idea

**Comment by optimumtact** [comment]
https://www.youtube.com/watch?v=Rnyb5O9haOE make it makes this sound

**Comment by idontwannasignup** [comment]
Maybe reflecting lasers is too much.

**Comment by PKPenguin321** [comment]
yeah that's why it's admin only and planned to only reflect thrown stuff gosh

also i updated the video

**Comment by Jordie0608** [comment]
Can't someone just charge a homerun and run around with it until they need it?

**Comment by PKPenguin321** [comment]
Yeah
I guess I could add a timer that makes it expire

**Comment by Jalleo** [comment]
Or make it so you have to stand still.

**Comment by Shapsy** [comment]
Make homeruns require that you aim for the mouth on help intent, like a baseball version of executions with guns

**Comment by GunHog** [comment]
Reminds me of this: https://www.youtube.com/watch?v=fvmifSyRxg8

File: code/game/objects/items/weapons/weaponry.dm [hunk 1]
```diff
@@ -320,11 +320,58 @@
...
+	if(!homerun_able)
+		..()
+		return
+	if(homerun_ready)
+		user << "<span class='notice'>You're already ready to do a home run!</span>"
+		..()
+		return
+	user << "<span class='warning'>You begin gathering strength...</span>"
+	playsound(get_turf(src), 'sound/magic/lightning_chargeup.ogg', 65, 1)
+	if(do_after(user, 90, target = src))
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/weaponry.dm]: Not sure this is going to work; target may need to be user to get the bar to show up.

**Comment by Jordie0608** [comment]
Probably should've waited for https://github.com/tgstation/tgstation/pull/19713#issuecomment-237763758

**Comment by PKPenguin321** [comment]
¯\\\_(ツ)_/¯ it works well enough

**Pull request merged**